app/javascript/app/components/stories/stories-styles.scss

```diff
@@ -77,17 +77,41 @@
 }
 
 .grid {
-  @include xy-grid($direction: 'vertical', $wrap: true);
+  @include xy-grid($wrap: true);
 
   height: 100%;
 
+  @media #{$tablet-landscape} {
+    @include xy-grid($direction: 'vertical', $wrap: true);
+  }
+
   > a {
-    @include xy-cell(3, $gutters: 0);
-    @include xy-cell(50%, $gutters: 0, $vertical:true);
+    @include xy-cell(6, $gutters: 0);
+    @include xy-cell(25%, $gutters: 0, $vertical:true);
+
+    @media #{$tablet-portrait} {
+      @include xy-cell(6, $gutters: 0);
+      @include xy-cell(33.333333%, $gutters: 0, $vertical:true);
+    }
+
+    @media #{$tablet-landscape} {
+      @include xy-cell(3, $gutters: 0);
+      @include xy-cell(50%, $gutters: 0, $vertical:true);
+    }
   }
 
   & :first-child {
-    @include xy-cell(6, $gutters: 0);
-    @include xy-cell(100%, $gutters: 0, $vertical:true);
+    @include xy-cell(12, $gutters: 0);
+    @include xy-cell(50%, $gutters: 0, $vertical:true);
+
+    @media #{$tablet-portrait} {
+      @include xy-cell(12, $gutters: 0);
+      @include xy-cell(33.333333%, $gutters: 0, $vertical:true);
```

Comment:
Arent we trying to avoid this? I think 4 of 12 would be better. It would be great to keep all the measures related to the 12 columns

Reply:
The problem I found is that the grid created measures have only a couple of decimal places and hence the cells do not span in the 100% of the container resulting in white gaps between them :(